Benqi-Agents/Qi-Transfers/src/utils.ts

```diff
@@ -0,0 +1,52 @@
+import { Finding, FindingSeverity, FindingType, LogDescription } from "forta-agent";
+import { ethers, BigNumber } from "ethers";
+
+export const EVENT_ABI = ["event Transfer(address indexed from, address indexed to, uint256 amount)"];
+export const BALANCE_ABI = ["function balanceOf(address account) external view returns (uint)"];
+export const benqiInterface = new ethers.utils.Interface(BALANCE_ABI);
+
+const isTestnet: boolean = false;
+
+export const QI_TOKEN_CONTRACT = isTestnet
+  ? "0x749590F04a3aa91B53E03124FDa4Ec0391D4Dd4E" // AVAX Testnet QI Token Contract address
+  : "0x8729438EB15e2C8B576fCc6AeCdA6A148776C0F5"; // AVAX Mainnet QI Token Contract address
+
+const QI_DECIMALS: number = 18;
+const TOTAL_SUPPLY: BigNumber = BigNumber.from("7200000000").mul(`${10 ** QI_DECIMALS}`);

```

Comment:
I also thought doing it that way. But then I decided that since it's a specific number that will never change, I wanted to avoid any possible problems while fetching the total supply data. still not very sure which one makes more sense.

Reply:
You did well, that value is constant in the contract.